src/main/java/com/happeningnow/Model/Category.java

```diff
@@ -0,0 +1,55 @@
+package com.happeningnow.Model;
+
+import jakarta.persistence.*;
+import org.hibernate.annotations.UuidGenerator;
+
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+@Table(name = "category")
+public class Category {
+    @Id
+    @UuidGenerator
+    @Column(name = "category_id")
+    private UUID categoryId;
+    @Column(nullable = false)
+    private String name;
+    @Column(nullable = false)
+    private String description;
+    @ManyToMany(mappedBy = "category")
+    private List<Event> event;
```

Comment:
It should be "eventList" since it is a list

Reply:
ok